### DISCUSSION TITLE
Grounds Rules Pt1

### DISCUSSION THREAD

**Discussion by crispybacon609** [crispybacon609/FamanaEU4#2] [Announcements]
Talk here thanks. I will lay out ground rules for the Git hub

I recommend you download the desktop version too

Firstly we will have a master copy. For right now that will be Adartho's version plus the new terrain and normal map.

If you want to make changes DO NOT make changes to the master till you have approval from a developer preferably me or Adartho.
If you run into a bug add it too the issues tab with details on how to recreate it. Will add more rules later

**Comment by u1145** [comment]
:trollface:

**Reply by Cilria** [reply]
Hopefully. we can resolve everything soon!
